### PR TITLE
Translated food description + shift-only and command-only description

### PR DESCRIPTION
**Pull request by Max094Reikeb**
This PR :
- Makes description automatically externalized (so, translatable)
- Adds a Shift-only description with checkbox
- Adds a Command-only/Control-only description with checkbox

It does the same as #12 and #20 but for food instead of blocks or items.

### CODE REVIEW

File: src/main/java/net/mcreator/ui/modgui/ItemGUI.java [hunk 1]
```diff
@@ -207,8 +207,12 @@ public ItemGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		hasGlow.setOpaque(false);
 		hasGlow.setSelected(false);
```
Comment (Goldorion) [inline src/main/java/net/mcreator/ui/modgui/ItemGUI.java:209]: Why did you add these lines for the item element?

Comment (Max094Reikeb) [reply src/main/java/net/mcreator/ui/modgui/ItemGUI.java:209]: I did not? There aren't any `+` or `-` to say it's a modified line...

File: src/main/java/net/mcreator/ui/modgui/ItemGUI.java [hunk 2]
```diff
@@ -207,8 +207,12 @@ public ItemGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		hasGlow.setOpaque(false);
 		hasGlow.setSelected(false);
 
+		onShiftOnly.setEnabled(true);
```
Comment (Goldorion) [inline src/main/java/net/mcreator/ui/modgui/ItemGUI.java:210]: Why did you add these lines to the item element?

File: src/main/java/net/mcreator/ui/modgui/ItemGUI.java (hunk 1 above)
Comment (Goldorion) [reply src/main/java/net/mcreator/ui/modgui/ItemGUI.java:209]: Wrong lines, sorry

File: src/main/java/net/mcreator/ui/modgui/ItemGUI.java (hunk 2 above)
Comment (Max094Reikeb) [reply src/main/java/net/mcreator/ui/modgui/ItemGUI.java:210]: Well because I forget about those. We need to set the checkboxes as enabled.
